web-utils/build.gradle.kts

```diff
@@ -70,18 +72,47 @@ repositories {
 }
 
 publishing {
-    repositories.maven("https://api.bintray.com/maven/icerockdev/backend/web-utils/;publish=1") {
-        name = "bintray"
+    repositories.maven("https://s01.oss.sonatype.org/service/local/staging/deploy/maven2/") {
+        name = "OSSRH"
 
         credentials {
-            username = System.getProperty("BINTRAY_USER")
-            password = System.getProperty("BINTRAY_KEY")
+            username = System.getProperty("OSSRH_USER")
+            password = System.getProperty("OSSRH_KEY")
         }
     }
     publications {
         register("mavenJava", MavenPublication::class) {
+            val projectName = project.name
             from(components["java"])
             artifact(sourcesJar.get())
+            pom {
+                name.set(projectName.replace("-", " ").capitalize())
+                description.set(name)
+                url.set("https://github.com/icerockdev/$projectName")
+                licenses {
+                    license {
+                        url.set("https://github.com/icerockdev/$projectName/blob/master/LICENSE.md")
+                    }
+                }
+
+                developers {
```

Comment:
here we can write multiple developers, so i think will be good to set here all our maintainers of this library :)